Makefile

```diff
@@ -130,3 +130,8 @@ clean:
 	rm -rf $(wildcard src/*/*/target) \
 	rm -rf $(wildcard src/*/*/*/target) \
 	rm -rf $(wildcard src/*/*/*/*/target)
+
+
+sendlocalcoverall:
+	cargo tarpaulin --out Xml --output-dir ./coverall
+	curl -s "https://codecov.io/bash" | bash -s
```

Comment:
This cannot be used locally. Please try it to see what happens. It can be removed.

Instead, define a `coverage` target, which is similar to the `test` target, but should run the `coverage` task in each of the directories.

Reply:
I do not really see how you would like I use the 
`
uses: codecov/codecov-action@v2.1.0
`

in the Makefile.

Reply:
That is a different thing. The _sending_ of the report does not belong in `Makefile`. What goes here is _generating_ it, i.e., invoking `coverage` for the individual directories, and there needs to be a corresping target that calls tarpaulin in `Makefile.inc`.
Please look at the `test` target for reference.